src/build/stable/utils/log_global_dependencies.ts

```diff
@@ -5,19 +5,67 @@ import { execSyncPretty } from './exec_sync_pretty';
 import { AZLE_PACKAGE_PATH } from './global_paths';
 
 export async function logGlobalDependencies(): Promise<void> {
-    const wasiVersion = execSyncPretty('cargo install --list | grep wasi2ic');
-    const nodeVersion = execSyncPretty('node --version');
-    const rustVersion = execSyncPretty('rustc --version');
+    const wasiVersion = await getWasiVersion();
+    const nodeVersion = await getNodeVersion();
+    const rustVersion = await getRustVersion();
 
-    const globalDependencies =
-        `wasi2ic version: ${wasiVersion}` +
-        `\n` +
-        `node version: ${nodeVersion}` +
-        `\n` +
-        `rustc version: ${rustVersion}`;
+    const globalDependencies = {
+        dependencies: {
+            wasi2ic: wasiVersion,
+            node: nodeVersion,
+            rustc: rustVersion
+        }
+    };
 
     await outputFile(
-        join(AZLE_PACKAGE_PATH, 'global_dependencies'),
-        globalDependencies
+        join(AZLE_PACKAGE_PATH, 'global_dependencies.json'),
+        JSON.stringify(globalDependencies, null, 2) // Format JSON with 2 spaces indentation
```

Comment:
Please format with 4 space indentation...I think that's what we are using in our prettier settings...did you have another reason for this?